src/app/Http/Controllers/Auth/VerifyEmailController.php

```diff
@@ -0,0 +1,60 @@
+<?php
+
+namespace Backpack\CRUD\app\Http\Controllers\Auth;
+
+use Backpack\CRUD\app\Http\Requests\EmailVerificationRequest;
+use Illuminate\Http\Request;
+use Illuminate\Routing\Controller;
+use Prologue\Alerts\Facades\Alert;
+
+class VerifyEmailController extends Controller
+{
+    public null|string $redirectTo = null;
+
+    /**
+     * Create a new controller instance.
+     *
+     * @return void
+     */
+    public function __construct()
+    {
+        $this->middleware(backpack_middleware());
+        $this->middleware('signed')->only('verifyEmail');
```

Comment:
Maybe we should call the signed middleware by class name, instead of alias? That way we don't need the fix in BackpackServiceProvider?

Reply:
We shouldn't. I mean, kernel.php is userland, they may have a `MySigned extends LaravelSigned` and change it in their kernel. 
We should keep using the alias as Laravel instructs users to do.

Reply:
Ok then... we can throw a custom error if the middleware we need aren't registered? When they're needed? Would that be better than the error Laravel throws?